mautrix_instagram/user.py

```diff
@@ -707,13 +719,16 @@ async def _sync(self, increment_total_backfilled_portals: bool = False) -> None:
         elif sync_count < 0:
             local_limit = None
 
-        await self._sync_threads_with_delay(
-            self.client.iter_inbox(
-                self._update_seq_id_and_cursor, start_at=resp, local_limit=local_limit
+        await self.proxy_with_retry(
+            "user._sync",
+            lambda: self._sync_threads_with_delay(
+                self.client.iter_inbox(
```

Comment:
Maybe all the retries should just be in mauigpapi's `std_http_post`/`_get` instead of each call separately 🤔

Reply:
> Maybe all the retries should just be in mauigpapi's std_http_post/_get instead of each call separately 🤔

Not sure why I didn't think of that, makes much more sense.